ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/SandboxIndexService.scala

```diff
@@ -235,28 +151,27 @@ class SandboxIndexService(
               case Some(LedgerOffset.Absolute(end)) =>
                 stream
                   .takeWhile(
-                    { item =>
-                      //note that we can have gaps in the increasing offsets!
-                      (item.offset.toLong + 1) < end.toLong //api offsets are +1 compared to backend offsets
+                    {
+                      case (offset, _) =>
+                        //note that we can have gaps in the increasing offsets!
+                        (offset + 1) < end.toLong //api offsets are +1 compared to backend offsets
                     },
                     inclusive = true // we need this to be inclusive otherwise the stream will be hanging until a new element from upstream arrives
                   )
-                  .filter(_.offset.toLong < end.toLong)
+                  .filter(_._1 < end.toLong)
             }
             // we MUST do the offset comparison BEFORE collecting only the accepted transactions,
```

Comment:
Please update the comment with the changes (e.g. the concept of "accepted transaction" doesn't exist anymore).